src/features/members/MemberList.jsx

```diff
@@ -11,31 +13,60 @@ export default function MemberList({ cohort }) {
   }
 
   const memberData = cohortMembers.map(member => ({
+    id: member.id,
     email: member.email,
-    status: member.user ? 'Registered' : 'Invited',
+    status: COHORT_MEMBERSHIP_STATUS_TYPE_MAP[member.status],
+    active: member.active,
   }));
 
+  const memberActionRows = memberData.reduce((componentList, member) => {
+    componentList[member.id] = <MemberActionRow member={member} cohort={cohort} />;
+    return componentList;
+  }, {});
+
+  const statusFilterChoices = [
+    {
+      name: COHORT_MEMBERSHIP_STATUS_TYPE_MAP[COHORT_MEMBERSHIP_STATUS.activated],
+      value: COHORT_MEMBERSHIP_STATUS_TYPE_MAP[COHORT_MEMBERSHIP_STATUS.activated],
+    },
+    {
+      name: COHORT_MEMBERSHIP_STATUS_TYPE_MAP[COHORT_MEMBERSHIP_STATUS.invited],
+      value: COHORT_MEMBERSHIP_STATUS_TYPE_MAP[COHORT_MEMBERSHIP_STATUS.invited],
+    },
+    {
+      name: COHORT_MEMBERSHIP_STATUS_TYPE_MAP[COHORT_MEMBERSHIP_STATUS.deactivated],
+      value: COHORT_MEMBERSHIP_STATUS_TYPE_MAP[COHORT_MEMBERSHIP_STATUS.deactivated],
+    },
+  ];
+
   return (
     <section className="mt-3">
       <DataTable
         isFilterable
         isPaginated
         isSortable
+        initialState={{ hiddenColumns: ['id', 'active'] }}
         itemCount={cohortMembers.length}
         defaultColumnValues={{ Filter: TextFilter }}
-        numBreakoutFilters={2}
+        numBreakoutFilters={1}
         data={memberData}
         columns={[
+          { Header: 'Membership ID', accessor: 'id', disableFilters: true },
           { Header: 'Email', accessor: 'email' },
+          { Header: 'Active', accessor: 'active', disableFilters: true },
           {
             Header: 'Status',
             accessor: 'status',
-            Filter: DropdownFilter,
+            Filter: CheckboxFilter,
             filter: 'includesValue',
-            filterChoices: [
-              { name: 'Registered', value: 'Registered' },
-              { name: 'Invited', value: 'Invited' },
-            ],
+            filterChoices: statusFilterChoices,
+          },
+        ]}
+        additionalColumns={[
+          {
+            id: 'action',
+            Header: 'Action',
+            Cell: ({ row }) => (memberActionRows[row.values.id]),
```

Comment:
Edge-case: updating status of a member while a filter is applied will reset the data in the table so it is no longer filtered, but the filter selections will still be checked. To reset the filters, you have to click the checked filter, then click Clear Filters.

I haven't figured out a way around this, and am not sure if it is a bug in Paragon or how data is handled in this component. I tried memoizing `memberData` to no change.

Reply:
I'm thinking it is probably a Paragon issue due to the similar behavior [noted in this QA card](https://trello.com/c/WC9JJLov/576-molp-admin-dashboard-clicking-clear-filters-doesnt-reset-the-selected-cohort-from-the-dropdown-menu-so-it-will-look-like-a-filte) but I will dig a bit more and see if I can find the source.